frida_tools/tracer.py

```diff
@@ -63,6 +64,10 @@ def _initialize(self, parser, options, args):
             self._targets = None
             self._profile = self._profile_builder.build()
             self._quiet = options.quiet
```

Comment:
Should only assign `self._output_path` here, setting `self._output` to `None`, and assign it in `_start()`.